javascript/react/src/components/TableCommands.tsx

```diff
@@ -63,11 +64,20 @@ export const TableCommands: React.FC<TableCommandsProps> = memo(
 					args: { to: selectedColumn, column: selectedColumn } as any,
 					input: inputTable?.id,
 				})
+				const id = createTableId(verb)
```

Comment:
Maybe do this before `readStep` and then include the id in the template? Seems weird to create and then mutate the id property

Reply:
omg so obvious, yes thankss